pkg/arm/v5/resources.go

```diff
@@ -316,7 +448,36 @@ func (g *simpleGenerator) nsgMaster() *network.SecurityGroup {
 func (g *simpleGenerator) nsgWorker() *network.SecurityGroup {
 	return &network.SecurityGroup{
 		SecurityGroupPropertiesFormat: &network.SecurityGroupPropertiesFormat{
-			SecurityRules: &[]network.SecurityRule{},
+			SecurityRules: &[]network.SecurityRule{
+				{
+					SecurityRulePropertiesFormat: &network.SecurityRulePropertiesFormat{
+						Description:              to.StringPtr("Allow HTTP traffic"),
+						Protocol:                 network.SecurityRuleProtocolTCP,
+						SourcePortRange:          to.StringPtr("*"),
+						DestinationPortRange:     to.StringPtr("80"),
+						SourceAddressPrefix:      to.StringPtr("Internet"),
+						DestinationAddressPrefix: to.StringPtr("*"),
```

Comment:
Would like to be able to set this to the IP address of the frontend load balancer.  Any ideas on how to accomplish this?